src/tests/ftest/util/general_utils.py

```diff
@@ -1404,6 +1405,19 @@ def get_journalctl(hosts, since, until, journalctl_type):
     return get_host_data(hosts=hosts, command=command, text="journalctl", error=err)
 
 
+def journalctl_time(when=None):
+    # pylint: disable=wrong-spelling-in-docstring
+    """Get time formatted for journalctl.
+
+    Args:
+        when (datetime, optional): time to format. Defaults to datetime.now()
```

Comment:
What is the format for when? Is it some number?

Reply:
> when (datetime, optional)

It's a `datetime` object. E.g.
```
>>> from datetime import datetime
>>> datetime.now()
datetime.datetime(2023, 6, 2, 18, 17, 2, 432460)
```